ggml/src/ggml-vulkan/ggml-vulkan.cpp

```diff
@@ -11379,7 +11379,7 @@ static void ggml_vk_check_results_0(ggml_backend_vk_context * ctx, ggml_cgraph *
             tensor_clone = ggml_cpy(ggml_ctx, src_clone[0], src_clone[1]);
         }
     } else if (tensor->op == GGML_OP_SET_ROWS) {
-        tensor_clone = ggml_set_rows(ggml_ctx, src_clone[0], src_clone[1]);
+        tensor_clone = ggml_set_rows(ggml_ctx, src_clone[0], src_clone[1], src_clone[2]);
```

Comment:
I'm not sure what the parameters should be here. Do we need to clone the dst tensor and pass that as the 'a' parameter?

Reply:
I didn't look at it in detail, but this fixed the issue I had with it.

Reply:
This is a weird operation because it doesn't keep around it's original 'a' parameter, and I think src_clone[2] will be null. And since it only writes sparsely to the destination we probably need to dup the original contents. So maybe something like:

```
   tensor_clone = ggml_dup(ggml_ctx, dst->view_src);
   tensor_clone = ggml_set_rows(ggml_ctx, tensor_clone, src_clone[0], src_clone[1]);
```

Reply:
I checked again, my code just fixed the compile problem, it does not work if actually used. But neither does yours, since the tensor data is still on GPU, so dup does not work. I think manually copying the view tensor to CPU and using it as view_src would work, but that would mean reworking the check_result functions to support this.

I just deactivated SET_ROWS for now to fix the issue, if we need set_rows checks they can be readded at a later point.